lms/djangoapps/instructor/services.py

```diff
@@ -31,7 +31,7 @@ def delete_student_attempt(self, student_identifier, course_id, content_id):
             - unique_student_identifier is an email or username
             - course_id is the id for the course
         """
-        course_id = SlashSeparatedCourseKey.from_deprecated_string(course_id)
+        course_id = CourseKey.from_string(course_id)
         student = get_student_from_identifier(student_identifier)
         try:
             module_state_key = course_id.make_usage_key_from_deprecated_string(content_id)
```

Comment:
@chrisndodge I have not changed `make_usage_key_from_deprecated_string` to `make_usage_key`. `make_usage_key` accepts another parameter `block_name`. I was not sure about the block name parameter. Perhaps you can take a look into this.